<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="delTreeButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>340</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить дерево</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Добавление элемента</string>
    </property>
    <widget class="QPushButton" name="createTreeButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-1000</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>620</width>
      <height>580</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Удаление</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>вершину со значением</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinIndexBox_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-1000</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QPushButton" name="delItemButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Бинарное дерево поиска</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>АВЛ дерево</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="squereButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>310</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Возвести корень в квадрат</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>210</y>
      <width>191</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Поиск</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>вершину со значением</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Найти</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinIndexBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-1000</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Поиск</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
